python/deploy.py

```diff
@@ -46,8 +50,20 @@ def check_env():
         env.check_env_var('CAMUNDA_CLUSTER_REGION', False)
         env.check_env_var('CAMUNDA_CLUSTER_HOST', False)
         env.check_env_var('CAMUNDA_CLUSTER_PORT', False)
+        env.check_env_var('OAUTH2_TOKEN_URL', False)
+        env.check_env_var('OAUTH_SCOPE', False)
```

Comment:
Should this have a 2 in it?
```suggestion
        env.check_env_var('OAUTH2_SCOPE', False)
```